app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -4498,9 +4469,13 @@ private void setupVideoSurface() {
             surfaceHolderCallback = new SurfaceHolderCallback(context, simpleExoPlayer);
             binding.surfaceView.getHolder().addCallback(surfaceHolderCallback);
             final Surface surface = binding.surfaceView.getHolder().getSurface();
-            // initially set the surface manually otherwise
-            // onRenderedFirstFrame() will not be called
-            simpleExoPlayer.setVideoSurface(surface);
+            // ensure player is using an unreleased surface, which the surfaceView might not be
+            // when starting playback on background or during player switching
+            if (surface.isValid()) {
```

Comment:
@TiA4f8R This should fix the issue with starting livestream on background. Let me know if this works and if you want this in another PR instead.